docs/releases/3.0.txt

```diff
@@ -385,6 +385,10 @@ Miscellaneous
   now have the ``placeholder`` attribute, which mainly may require some
   adjustments in tests that compare HTML.
 
+* :func:`~django.utils.http.urlencode` now encodes iterable values as are when
```

Comment:
Think this should be `as is`, not `as are`.

Reply:
`values` is plural.

Reply:
Doesn't matter that it is plural. It is definitely not "as are". Your alternative will work, however.

See https://english.stackexchange.com/a/9380

Reply:
Interesting. “As is” there is meant as a legal term, rather than a contraction. That’s not what’s intended at all. I’ll adjust. Good one. 👍

Reply:
Yes, it is strictly speaking a legal term, but we native speakers do use it generally too. Either alternative will be fine - it makes no sense *as is*. :grin: